cmd/vulcan-zap/main.go

```diff
@@ -153,14 +158,29 @@ func main() {
 			users.SetUserEnabled("1", "0", "True")
 		}
 
+		if opt.OpenapiUrl != "" {
+			_, err = client.Openapi().ImportUrl(opt.OpenapiUrl, opt.OpenapiHost)
+			if err != nil {
+				return fmt.Errorf("error importing openapi url: %w", err)
+			}
+		}
+
 		_, err = client.Pscan().DisableScanners(disabledScanners)
 		if err != nil {
 			return fmt.Errorf("error disabling scanners for passive scan: %w", err)
 		}
 
-		logger.Printf("Running spider %v levels deep...", opt.Depth)
-
 		client.Spider().SetOptionMaxDepth(opt.Depth)
+		if err != nil {
```

Comment:
Before, we weren't checking the error. I think it is better indeed to check the error however, then you need to read it by adding `_, err =` before the call to the function, otherwise you are reusing "the latest error value".

Reply:
Absolutely, thanks!